frontend/src/locale/index.ts

```diff
@@ -120,6 +120,60 @@ Let's get in touch to find the best solution for your business' needs!`,
 
     "profile.welcome": "Ciao {name}!",
     "profile.myProfile": "My Profile",
+
+    "profile.myGrant": "My Grant",
+    "profile.myGrant.nextSteps": "Next steps",
+    "profile.myGrant.grantType": "Type of Grant",
+    "profile.myGrant.appliedFor": "Applied for",
+    "profile.myGrant.appliedFor.ticket": "Ticket",
+    "profile.myGrant.appliedFor.travel": "Travel",
+    "profile.myGrant.appliedFor.accommodation": "Accommodation",
+    "profile.myGrant.status": "Status",
+    "profile.myGrant.status.pending": "Pending",
+    "profile.myGrant.status.pending.nextSteps":
+      "Your application is under review. No further action is needed from you at this moment.",
+    "profile.myGrant.status.rejected": "Rejected",
+    "profile.myGrant.status.rejected.nextSteps":
+      "Unfortunately, your grant request has been rejected due to limited funds and a high number of applications. For details, check the email we sent you.",
```

Comment:
does the email contain information?

Reply:
mmm not really: https://github.com/pythonitalia/pycon/blob/main/email-templates/src/templates/grants-rejected.html

Reply:
then can we remove that phrase from the text? maybe something like?

> Unfortunately, your grant request has been rejected due to limited funds and a high number of applications. We hope you will be able to join us regardless!